packages/trpc/src/procedures/labels/getLabelsByUserId.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { trpcSetup, tearDown } from "../../testutils";
+import { prisma } from "@recipesage/prisma";
+import { User } from "@prisma/client";
+import type { CreateTRPCProxyClient } from "@trpc/client";
+import type { AppRouter } from "../../index";
+
+describe("getLabelsByUserId", () => {
+  let user: User;
+  let trpc: CreateTRPCProxyClient<AppRouter>;
+
+  beforeAll(async () => {
+    ({ user, trpc } = await trpcSetup());
+  });
+
+  afterAll(() => {
+    return tearDown(user.id);
+  });
+
+  describe("success", () => {
+    it("get label by userId", async () => {
+      const label = await prisma.label.create({
+        data: {
+          userId: user.id,
+          title: "meat",
+        },
+      });
+      const response = await trpc.labels.getLabelsByUserId.query({
+        userIds: [user.id],
+      });
+      expect(response[0].id).toEqual(label.id);
+    });
+  });
+  it("fails to get lebels with differrent user", async () => {
+    const { user: user2, trpc: trpc2 } = await trpcSetup();
```

Comment:
Probably best to create a second user inside of the test rather than doing a duplicate setup here